airbyte-integrations/connectors/source-bing-ads/source_bing_ads/manifest.yaml

```diff
@@ -144,8 +143,120 @@ definitions:
               - LastModifiedTime
             value: '{{ format_datetime(record.LastModifiedTime, "%Y-%m-%dT%H:%M:%S.%f" ) }}'
 
+  campaigns_stream:
+    type: DeclarativeStream
+    name: campaigns
+    primary_key: Id
+    schema_loader:
+      type: InlineSchemaLoader
+      schema:
+        $ref: "#/schemas/campaigns"
+    retriever:
+      type: SimpleRetriever
+      requester:
+        type: HttpRequester
+        url_base: https://campaign.api.bingads.microsoft.com
+        path: /CampaignManagement/v13/Campaigns/QueryByAccountId
+        http_method: POST
+        request_headers:
+          Content-Type: application/json
+          CustomerAccountId: "{{ stream_partition['account_id'][0] }}"
+          CustomerId: "{{ stream_slice.extra_fields['ParentCustomerId'][0] }}"
+          DeveloperToken: "{{ config['developer_token'] }}"
+          Authorization: "Bearer {{ config['_token']['access_token'] }}"
```

Comment:
you can define `authenticator: "#/definitions/authenticator"` so you don't need to overwrite  `Authorization` header here.